app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
```

Comment:
Code Quality Issue: It's good practice to annotate function parameters and return types. This helps others understand what type of parameters are expected and what type of value the function will return.